src/optimum/nvidia/models/mistral.py

```diff
@@ -74,6 +74,7 @@ def from_config(config: TransformersPretrainedConfig) -> "TensorRTConfig":
             max_lora_rank=64,
             head_size=config.hidden_size / config.num_attention_heads,
             quantization=qconfig,
+            rotary_base=config.rope_theta,
```

Comment:
Was a bug e.g. for https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.2/blob/main/config.json#L18 where rotary_base is not the default 10000.0